### DISCUSSION TITLE
Hydrate Lucid Model

### DISCUSSION THREAD

**Discussion by kush-prof** [adonisjs/core#2588] [Help] [answered]
Can I hydrate a Lucid model from the response of Database query method?

**Comment by RomainLanz** [comment, marked as answer]
Hey @kush-prof! 👋 

Yes, you can use the [`merge`](https://docs.adonisjs.com/reference/orm/base-model#merge) or the [`fill`](https://docs.adonisjs.com/reference/orm/base-model#fill) method of your model to hydrate it.
I would recommend to directly use the model to do any querying thought.

**Reply by kush-prof** [reply]
@RomainLanz Is there any documentation regarding these methods?

**Reply by RomainLanz** [reply]
Yes, you can click on their name in my previous message.

**Reply by kush-prof** [reply]
ok thanks
